int64 number
---
int64 squared_number